### PR TITLE
Fix/conflict

### PR DESCRIPTION
**Pull request by marinofaggiana**
Fix keep both conflict action

### CODE REVIEW

File: iOSClient/Main/Create cloud/NCCreateFormUploadConflict.swift [hunk 1]
```diff
@@ -290,30 +251,32 @@ extension NCCreateFormUploadConflictDelegate {
             if metadatasConflictNewFiles.contains(metadata.ocId) && metadatasConflictAlreadyExistingFiles.contains(metadata.ocId) {
```
Comment (thisIsTheFoxe) [inline iOSClient/Main/Create cloud/NCCreateFormUploadConflict.swift:251]: I'm not sure about the difference between `metadatasConflictNewFiles` and `metadatasConflictAlreadyExistingFiles`. Would you mind to explain what they are used for? Or better, add documentation for those.

**Review by thisIsTheFoxe — DISMISSED** [review]
Thanks, looks good already.
However, the problem is that all the files are not moved (only *MOV*s?). I'm still getting an error...

File: iOSClient/Main/Create cloud/NCCreateFormUploadConflict.swift [hunk 2]
```diff
@@ -156,66 +156,27 @@ extension NCCreateFormUploadConflictDelegate {
...
-                
-                self.delegate?.dismissCreateFormUploadConflict(metadatas: self.metadatasNOConflict)
-                
-            } else {
-                
-                self.appDelegate.networkingProcessUpload?.createProcessUploads(metadatas: self.metadatasNOConflict)
-            }
-            
-            self.dismiss(animated: true, completion: nil)
+            self.buttonContinueTouch(self)
```
[the remark this answers is not in the document]
Comment (marinofaggiana) [reply iOSClient/Main/Create cloud/NCCreateFormUploadConflict.swift:166]: feel free to edit

File: iOSClient/Main/Create cloud/NCCreateFormUploadConflict.swift [hunk 3]
```diff
@@ -156,66 +156,27 @@ extension NCCreateFormUploadConflictDelegate {
...
-                    }
-                }
+                self.metadatasConflictNewFiles.append(metadata.ocId)
             }
-            
-            if self.delegate != nil {
-                self.delegate?.dismissCreateFormUploadConflict(metadatas: self.metadatasNOConflict)
-            } else {
-                self.appDelegate.networkingProcessUpload?.createProcessUploads(metadatas: self.metadatasNOConflict)
+            for metadata in self.metadatasUploadInConflict {
```
[the remark this answers is not in the document]
Comment (marinofaggiana) [reply iOSClient/Main/Create cloud/NCCreateFormUploadConflict.swift:175]: hahaha no

**Comment by marinofaggiana** [comment]
@thisIsTheFoxe approve this

**Comment by thisIsTheFoxe** [comment]
Hi @marinofaggiana thanks for pinging me, I'll get to this as soon as I can.
But please note the comment of my last review. I can't see all the files being moved. I'm pretty sure that's still the case which would mean the original issue still persists.

**Review by thisIsTheFoxe — CHANGES_REQUESTED** [review]
Yes, as I thought.. the error is still there.
Please make sure the "keep both" option is working. For details see also #1742 
Simulator Screen Shot - iPhone 12 Pro Max - 2021-11-11 at 11 51 50

**Comment by marinofaggiana** [comment]
I have tested and works ... "keep both", are you sure of you use this PR ?

**Comment by thisIsTheFoxe** [comment]
Yes, but the issue is about *files* not media. E.g. a .PDF or .TXT file.

**Comment by marinofaggiana** [comment]
Ah. ok I think it's another source ...

**Review by thisIsTheFoxe — APPROVED** [review]

**Pull request merged**